src/assets/i18n/fr.json

```diff
@@ -1812,15 +1812,20 @@
       "invoices": "Factures"
     },
     "graph": {
-      "battery": "Niveau de Batterie (%)",
-      "power": "Puissance Instantanée (kW)",
-      "energy": "Energie Délivrée (kW.h)",
-      "reset_zoom": "Remise à Zéro",
-      "no_consumption": "Aucune donnée disponible",
-      "cumulated_amount": "Montant",
-      "limit_plan_watts": "Limitation du Plan (kW)",
-      "limit_watts": "Limitation Max (kW)",
-      "load_all_consumptions": "Charger toutes les valeurs de consommation (pas d'optimisation)"
+      "battery": "Battery Level (%)",
+      "power": "Instant Power (kW)",
+      "amps": "Instant Amps (A)",
+      "energy": "Energy Delivered (kW.h)",
+      "reset_zoom": "Reset",
+      "no_consumption": "No consumption data available",
+      "cumulated_amount": "Amount",
+      "limit_plan": "Plan Limit",
+      "limit": "Max Limit",
+      "limit_watts": "Max Limit (kW)",
+      "limit_amps": "Max Limit (A)",
+      "load_all_consumptions": "Load all consumption values (without optimization)",
+      "unit_kilowatts": "kW",
+      "unit_amperage":  "Amp"
```

Comment:
Just add the new entries to translate, do not delete the previous ones already correctly translated.